pkgs/by-name/ne/nerdctl/package.nix

```
@@ -54,7 +68,22 @@ buildGoModule rec {
     description = "Docker-compatible CLI for containerd";
     mainProgram = "nerdctl";
     license = lib.licenses.asl20;
-    maintainers = with lib.maintainers; [ developer-guy jk ];
-    platforms = lib.platforms.linux;
+    maintainers = with lib.maintainers; [
+      developer-guy
+      jk
+    ];
+    platforms = [
+      "x86_64-linux"
+      "i686-linux"
+
+      "aarch64-linux"
+      "armv7l-linux"
+
+      "powerpc64le-linux"
+      "riscv64-linux"
+      "s390x-linux"
+
+      "x86_64-freebsd"
+    ];
```

```suggestion
    platforms = lib.platforms.linux ++ [ "x86_64-freebsd" ];
```

Is this the intent of the change? Seems cleaner and would be a separate commit for changing platform support.